tests/test_triton_dropout.py

```diff
@@ -88,14 +88,16 @@ def test_dropout(shape, amp, bias, p):
         x_ref = (x + b if bias else x).to(y.dtype)
         assert torch.allclose(x_ref, y, rtol=tol), f"{x[x>y]}"
 
-        # Check that 1 means dropout for sure
-        y = triton_dropout(x, p=1, bias=b)
```

Comment:
minor change, p=1 means nothing out, which does not make a lot of sense (technically we should skip the kernel and return torch.zeros_like()..)

Reply:
fixed, I brought both the test and implementation back